staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -115,73 +136,3 @@ func (a customResourceDefinitionStorageStrategy) MatchCustomResourceDefinitionSt
 		GetAttrs: a.GetAttrs,
 	}
 }
-
-type customResourceValidator struct {
```

Comment:
hard to follow changes combined with the move... can you keep it in place or separate the straight move into a separate commit?

Reply:
can keep it in place. the move is in the subresources PR (adds lots of things there).